ports/casadi/portfile.cmake

```
@@ -0,0 +1,39 @@
+vcpkg_check_linkage(ONLY_DYNAMIC_LIBRARY)
+
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO casadi/casadi
+    REF "${VERSION}"
+    SHA512 2c95368281f0bda385c6c451e361c168589f13aa66af6bc6fadf01f899bcd6c785ea7da3dee0fb5835559e58982e499182a4d244af3ea208ac05f672ea99cfd1
+    HEAD_REF main
+)
+
+# Pending upstream fix https://github.com/casadi/casadi/issues/3896
+set(VCPKG_POLICY_SKIP_ABSOLUTE_PATHS_CHECK enabled)
+
+vcpkg_cmake_configure(
+    SOURCE_PATH "${SOURCE_PATH}"
+	OPTIONS
+	 -DWITH_SELFCONTAINED=OFF
+	 -DWITH_TINYXML=OFF
+	 -DWITH_BUILD_TINYXML=OFF
+	 -DWITH_QPOASES=OFF
+	 -DWITH_SUNDIALS=OFF
+	 -DWITH_CSPARSE=OFF
+	 -DLIB_PREFIX:PATH="lib"
+	 -DBIN_PREFIX:PATH="bin"
+	 -DINCLUDE_PREFIX:PATH="include"
+	 -DCMAKE_PREFIX:PATH="share/${PORT}"

```

```suggestion
	 "-DLIB_PREFIX:PATH=lib"
	 "-DBIN_PREFIX:PATH=bin"
	 "-DINCLUDE_PREFIX:PATH=include"
	 "-DCMAKE_PREFIX:PATH="share/${PORT}"
```
(Quotes are not really needed here. But if you want to apply them, put it around the whole option. CMake language.)